src/Service/QueueManager.php

```diff
@@ -218,7 +218,28 @@ private function jobExists(string $jobUuid): Promise
     private function processBatch(): \Generator
     {
         $this->logger->debug('Processing batch');
-        yield $this->elasticSearch->bulkIndexJobs($this->batch, $this->flowConfig->getTube());
+        $result = yield $this->elasticSearch->bulkIndexJobs($this->batch, $this->flowConfig->getTube());
+
+        if ($result['errors'] === true) {
+            foreach ($result['items'] as $item) {
+                if (!array_key_exists('index', $item)) {
+                    $this->logger->error(
+                        'Unexpected response item in bulk index response',
+                        ['bulk_index_response_item' => $item]
+                    );
+                    continue;
+                }
+                $itemStatusCode = $item['index']['status'] ?? null;
+                if (!$this->isSuccessfulStatusCode($itemStatusCode)) {
+                    $uuid = $item['index']['_id'];
```

Comment:
Here we should check the item format. There could be no UUID in `$item['index']['_id']` (potentially with bulk API you could ask for different operations than `index` and maybe there might not be any `_id` for that operation).

In case of no UUID in `$item['index']['_id']` I think we should raise an error.